src/libraries/System.Text.Encodings.Web/tests/System.Text.Encodings.Web.Tests.csproj

```diff
@@ -44,15 +45,19 @@
     <Compile Include="UnicodeRangeTests.cs" />
     <Compile Include="UrlEncoderTests.cs" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
Maybe move to the end of the csproj. It is easier to have all the `Compile` items under the same `ItemGroup`. Or maybe just combine the `ItemGroup` after this one with the one above. It makes it more readable.